templates/js/tests/step_implementation.js

```diff
@@ -2,13 +2,13 @@
 "use strict";
 const { openBrowser,write, closeBrowser, goto, press,text, contains } = require('taiko');
 const assert = require("assert");
-const headless = process.env.headless_chrome.toLowerCase() == 'true' ? true : false;
+const headless = process.env.headless_chrome.toLowerCase() === 'true';
 
-beforeSuite(async () => openBrowser({ headless: headless }));
+beforeSuite(() => openBrowser({ headless: headless }));
```

Comment:
These functions has to be async so the it will wait for `openBrowser` to complete. May be making it use explicit await will be better.
```
beforeSuite(async () => {
      await openBrowser({ headless: headless })
});
```

Reply:
Actually
```js
beforeSuite(async () => {
   await openBrowser({ headless: headless })
});
```
and
```js
beforeSuite(() => openBrowser({ headless: headless }));
```
Has same return type: `Promise`

In first case we will give back promise which will be resolved when browser opened.
In second case we will give back browser opening promise.

Reply:
Sorry if I was not clear, in this particular case yes as there is only one statement but the aim of initialisation is to provide a sample that the user can build on, making it explicit will help reminding user to wait for further actions added.

Reply:
Ok, got it.

PR updated